WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -116,7 +116,7 @@ def post_announcement
     end
     title = "#{comp.name} on #{date_range_str} in #{comp.cityName}, #{comp.countryId}"
 
-    body = "The [#{comp.name}](#{root_url}results/c.php?i=#{comp.id})"
+    body = "The [#{comp.name}](#{root_url(locale:nil)}results/c.php?i=#{comp.id})"
```

Comment:
Exactly, because this part of the website was not on rails yet I did not translate it.
Although now that the competition page has been ported to rails I should rebase this work, and this link would surely become localized.